openapi.yaml

```diff
@@ -13073,8 +13079,11 @@ components:
       properties:
```

Comment:
There could be some objects that contain only one property - but which might not be foreign key - like this one, schema for updating a network, only has `name` as property that can be updated.
Could there be any unintended change in behaviour for these?

Reply:
Perhaps. In general we want objects to be nullable no matter how many children they have, but I didn't commit to that in this PR.

I've done some more thinking after your review. Let me rework the logic so that it only applies if there's a single key, and the key is `id`. That will scope to exactly what we're trying to achieve here.